src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -348,17 +358,22 @@ function MultiCombobox<Item>({
                     </Box>
                   </>
                 </Flex>
-                {isOpen && canClearAllAfter && value.length >= canClearAllAfter && (
-                  <AbstractButton
-                    width="100%"
-                    py={1}
-                    backgroundColor="blue-400"
-                    fontSize="2x-small"
-                    onClick={clearSelectedItems}
-                  >
-                    Clear All
-                  </AbstractButton>
-                )}
+                {isOpen &&
+                  canClearAllAfter &&
+                  value.length >= canClearAllAfter &&
+                  (renderClearAll ? (
+                    renderClearAll(clearSelectedItems)
```

Comment:
I don't think that we have to use two different styles of the "clear selection"  button. I believe that we should just replace the old button style with the new one.

Reply:
Kind of agree here

Reply:
ill hit up amy as well